action.yml

```diff
@@ -54,6 +54,10 @@ inputs:
       [Optional] create target branch if not exist. Defaults to `false`
     default: false
     required: false
+  exclude-directories:
+    description: '[Optional] The directory to exclude for remove from remote repo'
```

Comment:
This could probably be changed to say something like 

```sh
ignore-directories:
  description: '[Optional] whitespace delimited set of directories to ignore.'
```

It feels like a different naming makes it more obvious that the directories will be ignored in the overall process.